cypress/support/index.ts

```diff
@@ -0,0 +1,42 @@
+type CustomCypressCommands = Record<string, (...args: any[]) => Cypress.CanReturnChainable>;
+
+/**
+ * Returns an object that contains custom Cypress commands with correct return types
+ * (Cypress.Chainable)
+ */
+export type GetCypressChainableFromCommands<CustomCommands extends CustomCypressCommands> = {
+    [P in keyof CustomCommands]: void extends ReturnType<CustomCommands[P]>
+        ? (...args: Parameters<CustomCommands[P]>) => Cypress.Chainable
+        : CustomCommands[P];
+};
+
+declare global {
+    namespace Cypress {
+        // eslint-disable-next-line @typescript-eslint/no-empty-interface
+        export interface Chainable extends GetCypressChainableFromCommands<typeof commands> {}
+    }
+}
+
+export const addCommands = (commands: CustomCypressCommands) =>
+    Object.entries(commands).forEach(([name, command]) => {
+        Cypress.Commands.add(name, command);
+    });
+
+const commands = {
+    getAdminToken: () =>
```

Comment:
Is this command visible in stage's `cy`? I have not checked, I'm just asking to be sure that extending the namespace in a separate package worked :smile:

Reply:
And for extra effort: does jumping to the function definition work too, so we see the source code in `node_modules`?

Reply:
Nevermind, I verified it :smile: It's working flawlessly when we 

```ts
import 'cloudify-ui-common/cypress/support';
```

so, good job! :smile: